integration/container_log_test.go

```diff
@@ -66,9 +126,9 @@ func TestLongContainerLog(t *testing.T) {
 	longLineCmd := fmt.Sprintf("i=0; while [ $i -lt %d ]; do printf %s; i=$((i+1)); done", maxSize+1, "c")
 	cnConfig := ContainerConfig(
 		containerName,
-		"busybox",
+		testImage,
 		WithCommand("sh", "-c",
-			fmt.Sprintf("%s; echo; %s; echo; %s", shortLineCmd, maxLenLineCmd, longLineCmd)),
+			fmt.Sprintf("%s; echo; %s; echo; %s; echo", shortLineCmd, maxLenLineCmd, longLineCmd)),
```

Comment:
maybe add a test to check processing of the carriage return processing '\r'

Reply:
That is for Windows, if I remember correctly.